plans/e2e/tier-2.fmf

```diff
@@ -0,0 +1,38 @@
+summary: qm tier-2 test plan
+
+discover:
+    how: fmf
+    filter: tag:tier-2
+
+environment+:
+  CONTROL_CONTAINER_NAME: host
+  FFI_SETUP_OPTIONS: none
+  PACKAGE: qm
+  PACKIT_COPR_PROJECT: release
+  SET_CS_REPO: yes
+  SKIP_EPEL_REPO: yes
+
+
+
+prepare+:
+  - name: Install podman
+    how: install
+    order: 20
+    package:
+      - podman
+
+  - name: Set QM environment
+    how: shell
+    order: 50
+    script: |
+        if ! stat /run/ostree-booted > /dev/null 2>&1 && \
+                [ "$(systemctl is-active qm)" != "active" ] ; then
+            cd tests/e2e
+            ./set-ffi-env-e2e "${FFI_SETUP_OPTIONS}"
+        fi
```

Comment:
I think this part could move to ./set-ffi-env-e2e script

Reply:
Hi @Yarboa , thanks for the review.
Should this part be deleted or should it be kept here and added it to `./set-ffi-env-e2e` as well?

Reply:
MHO, it should not be here, @dougsland WDYT?